lib/data/landing_page_content_items/landing_page.yaml

```diff
@@ -1,4 +1,21 @@
 blocks:
+- type: main_navigation
+  links:
+  - label: Lorem for Ipsum
+    active: true
+  - label: Ipsums for Lorem
+    link: /ipsum-2
+    active: false
+  - label: Our Lorem
+    link: /our-lorem
+    active: false
+    children:
```

Comment:
Having child menu items is really flexible, but is that a requirement? These child items aren't displayed on desktop either - how does that work? 

(sorry, feels like you've gone the extra mile here and I'm nitpicking)

Reply:
No worries @andysellick - The design seems to show child items on the mobile version, which don't exist on the desktop version. The implementation is quite simple, the child list elements just hide if the screen resolution is mobile. Do we need to ask someone about this functionality?